src/components/accessCounter.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import { ReactElement, useEffect, useState } from "react";
```

Comment:
ここの２行はまとめられると思います（私のコミットでtsconfigを書き換えたのでいけるはず）。
```suggestion
import React, { ReactElement, useEffect, useState } from "react";
```